src/utils/sounds.js

```diff
@@ -101,12 +111,12 @@ export const Sounds = {
 		if (playWaitingSound) {
 			await this.playWaiting()
 		} else {
-			this._playSounceOnce('LibremEmailNotification.ogg')
+			this._playSoundOnce('join_call.wav')
```

Comment:
/apps/spreed/img/join_call.wav:1 Failed to load resource: the server responded with a status of 404 (Not Found)
Uncaught (in promise) DOMException: Failed to load because no supported source was found.